kovri.md

```diff
@@ -17,17 +17,35 @@ Kovri is a free, decentralized, anonymity technology based on I2P's open specifi
 Kovri uses both encryption and sophisticated routing techniques to create a private overlay-network across the Internet.
 This protected overlay allows users to hide their geographical location and IP address.
 An IP address is like a home address for the Internet, so this is quite sensitive information, to say the least.
-Let's go through a few scenarios to see how this lightweight, security-focused software helps strengthen the unique privacy properties of Monero.
+
 
 ## 4) Examples
 
-Suppose Alice wants to send a transaction to Bob, and she begins the transaction.
-In order for her transaction to have the greatest chance of being included in the next block, she needs to reach as many nodes as possible.
-Alice connects to five nodes and these nodes then forward the transaction to other nodes, which in turn rebroadcast it repeatedly until many (or all) nodes in the network have this transaction.
-Unfortunately, Alice takes a risk when broadcasting her transaction.
-Some nodes may be logging IP addresses when transaction requests are received.
-Even though wallet addresses and the amount of Monero remain private, an attacker with enough resources could attempt to associate transactions with IP addresses to determine where transactions originate from.
-This could potentially lead to an attacker knocking on Alice's door or blocking her transaction from the rest of the network.
+Let's go through a few scenarios to see how this lightweight, security-focused software helps strengthen the unique privacy properties of Monero.
+
+Suppose Alice wants to send Monero to Bob.
+Alice's wallet creates a transaction, which then asks her daemon (her node) to broadcast it to the Monero network.
+Alice's node is connected to several nodes run by network peers, so those peers are first to receive a broadcast of her transaction.
+Those peers have peers of their own, to which they rebroadcast Alice's transaction.
+This process repeats until most - or all - nodes on the Monero network have Alice's transaction.
```

Comment:
Lines 28-30 are too wordy... could be rephrased into a single sentence.

Reply:
I would like it to be less wordy as well.  I'm not sure how to clearly state it, though, in fewer words. I tried to be as brief as possible.  Anyway, suggestions are welcome.